### PR TITLE
CEDS-1733 Timers on upstream and hot mongo access path

### PR DESCRIPTION
**Pull request by bambuchaAdm**

### CODE REVIEW

File: test/unit/uk/gov/hmrc/exports/base/MockMetrics.scala [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import com.kenshoo.play.metrics.Metrics
+
+trait MockMetrics {
+
+  val metricRegistry = new MetricRegistry()
+
+  val metrics: Metrics = new Metrics {
+    override def defaultRegistry: MetricRegistry = metricRegistry
+
+    override def toJson: String = ???
```
Comment (MaciejRewera) [inline test/unit/uk/gov/hmrc/exports/base/MockMetrics.scala]: Is it intentional?

**Comment by bambuchaAdm** [comment]
test this please

File: test/unit/uk/gov/hmrc/exports/base/MockMetrics.scala (hunk 1 above)
Comment (bambuchaAdm) [reply test/unit/uk/gov/hmrc/exports/base/MockMetrics.scala]: Yes - this is used by framework for reporting and in client code is unused - maybe i should make little scaladoc about this ?

**Comment by bambuchaAdm** [comment]
test this please

**Pull request merged**